app/api/routers/breeze_buddy/data_sources/__init__.py

```
@@ -0,0 +1,146 @@
+)
+
+router = APIRouter()
+
+
+# ─── Discovery routes (must come before /{id}) ────────────────────────────────
+
+
+@router.get("/data-sources/sheets/tabs", response_model=TabsResponse)
+async def get_sheet_tabs(
+    spreadsheet_url: str = Query(..., description="Full Google Sheets URL"),
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)